.gitignore

```diff
@@ -0,0 +1,68 @@
+# Xcode
+#
+# gitignore contributors: remember to update Global/Xcode.gitignore, Objective-C.gitignore & Swift.gitignore
+
+## Build generated
+build/
+DerivedData/
+
+## Various settings
+*.pbxuser
+!default.pbxuser
+*.mode1v3
+!default.mode1v3
+*.mode2v3
+!default.mode2v3
+*.perspectivev3
+!default.perspectivev3
+xcuserdata/
+# .DS_Store
+# */xcuserdata/*
```

Comment:
Why are these commented (lines 19 and 20). We want them ignored from the repo.
I still see these files in your PR, we can't merge them into master.